lib/my-bundle-1.0.00.bundle

```diff
@@ -83,5 +132,15 @@
         <l7:Mapping action="NewOrExisting" srcId="b24e8f6047856b780f8237ff36fc3714" type="FOLDER"/>
         <l7:Mapping action="NewOrExisting" srcId="b24e8f6047856b780f8237ff36fc3713" type="POLICY"/>
         <l7:Mapping action="NewOrExisting" srcId="b24e8f6047856b780f8237ff36fc3718" type="ENCAPSULATED_ASSERTION"/>
+        <l7:Mapping action="NewOrExisting" srcId="c3e01a74e06baddb76b047986c280d02" type="SERVICE">
```

Comment:
Service mapping was missing so it wasn't being created.

Reply:
hmm.. thats correct..Dint realize I missed it.